routing/bandwidth.go

```diff
@@ -180,11 +196,11 @@ func (b *bandwidthManager) getBandwidth(cid lnwire.ShortChannelID,
 		return 0
 	}
 
-	// If our link isn't currently in a state where it can add
-	// another outgoing htlc, treat the link as unusable.
-	if err := link.MayAddOutgoingHtlc(amount); err != nil {
+	// If our link isn't currently in a state where it can add another
+	// outgoing htlc, treat the link as unusable.
+	if err := link.MayAddOutgoingHtlc(htlcAmount); err != nil {
 		log.Warnf("ShortChannelID=%v: cannot add outgoing "+
-			"htlc: %v", cid, err)
+			"htlc with amount %v: %v", cid, htlcAmount, err)
```

Comment:
oh so previously we would only be able to route as much value as the local balance itself, e.g 100k sats or w/e, as this call would  always return err for higher amounts